install/terraform/open-match-build/README.md

```diff
@@ -0,0 +1,22 @@
+# Open Match Continuous Integration
+
+Open Match runs it's CI in the open-match-build GCP project.
+We primarily use Google Cloud Build to orchestrate our builds and the artifacts are published to GCS and gcr.io/open-match-build.
+
+These Terraform templates codify the infrastructure required to perform CI runs.
+
+```bash
```

Comment:
Are these currently intended only for CIs? Or do we expect demos etc., to use these? I see the project defaulted to "open-match-build" so currently if someone tries these steps manually - would that be a new cluster creation in the project? or would the state of CI cluster be syncd?

Reply:
This is solely for open-match-build. It'll eventually be possible for someone to take this template and recreate the Open Match CI build pipeline elsewhere. Basically all the tooling we have for this project will be open sourced and checked in.